module_evaluations/TCR-6_2022-09-28.MD

```diff
@@ -0,0 +1,61 @@
+* [ ] Environment vars are documented in the ModuleDescriptor - _note: read more at [https://wiki.folio.org/pages/viewpage.action?pageId=65110683](https://wiki.folio.org/pages/viewpage.action?pageId=65110683)_
+  * The following documented environment variables are not present in the module descriptor: `EVENTS_PUBLISHER_BOOTSTRAP_SERVERS`, `EVENTS_CONSUMER_BOOTSTRAP_SERVERS`, `EVENTS_PUBLISHER_ZK_CONNECT`, `EVENTS_CONSUMER_ZK_CONNECT`
+* [X] Back-end modules must define endpoints consumable by other modules in the Module Descriptor “provides” section
+* [ ] All API endpoints are documented in RAML or OpenAPI
+  * This work is being done by the developers, but it is not yet complete.
+* [X] All API endpoints protected with appropriate permissions
+* [X] No excessive permissions granted to the module
+* [X] Code of Conduct statement in repository
+* [ ] Installation documentation included
+  * Documentation which assits an implementor in getting this module runnin locally would satisfy this requirement
+* [X] Contribution guide is included in repo
+* [X] Module provides reference data (if applicable)
+* [X] Personal data form is completed, accurate, and provided as PERSONAL_DATA_DISCLOSURE.md file
+* [X] Sensitive information is not checked into git repository
+* [X] Module is written in a language and framework that FOLIO development teams are familiar with _e.g. Vertx/RMB, Spring Way/folio-spring-base, and React/Stripes_
+  * Groovy is already in use in other FOLIO modules
+* [X] Back-end modules are based on Maven/JDK 11 and provide a Dockerfile
+* [ ] Integration (API) tests written in Karate if applicable -_note: these tests are defined in https://github.com/folio-org/folio-integration-tests_
+  * It is important that the itegration test are written in a technology that can be integrated esily into FOLIO's CI pipeline.
+* [ ] Back-end unit tests at 80% coverage
+  * There is sufficient branching logic within this implementation to strongly suggest that unit testing would be advantageous
+  * Integration tests might be used as a proxy for Unit tests, but only if they can produce a coverage report showing that > 80% of the code has been executed during the testing
+  * Given the extent of generated code in this application, manual coverage reports showing a coverage in excess of 80% (on authored code) will accepted.
+* [X] Data is segregated by tenant at the storage layer
+* [X] Back-end modules don’t access data in DB schemas other than their own and public
+* [X] Tenant data is segregated at the transit layer
+* [X] Back-end modules respond with a tenant’s content based on x-okapi-tenant header
+* [ ] Standard GET /admin/health endpoint returning a 200 response -_note: read more at [https://wiki.folio.org/display/DD/Back+End+Module+Health+Check+Protocol](https://wiki.folio.org/display/DD/Back+End+Module+Health+Check+Protocol)_
+  * I did not find evidence that this module offers such an endpoint.
```

Comment:
This [code](https://github.com/folio-org/mod-oa/blob/ae611ab8fa284e1448adb51efc400b151451a848/service/grails-app/conf/application.yml#L116-L132) suggests that a health endpoint is provided by the underlying tooling (much like RMB)